functions/time.js

```diff
@@ -0,0 +1,7 @@
+export const noteDate = (time) => {
+  return new Date(time)
```

Comment:
Please use `Intl` API
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat

Reply:
Thanks for the tip.